### PR TITLE
SUBMARINE-876. Migrate the Unit tests from Travis CI to GitHub Actions

### PR DESCRIPTION
**Pull request by KUAN-HSUN-LI**
### What is this PR for?
Migrate the Unit tests from Travis CI to GitHub Actions.

### What type of PR is it?
[CI/CD]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-876

### How should this be tested?
GitHub Actions master workflow

### Screenshots (if appropriate)
image
image

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -161,6 +174,326 @@ jobs:
...
+        java-version: "1.8"
+    - name: Set up Maven 3.6.3
+      uses: stCarolas/setup-maven@v4
+      with:
+        maven-version: 3.6.3
+    - name: Check version
+      run: |
+        mvn --version
+        java -version
+    - name: Prepate default mysql-data
```
Comment (ByronHsu) [inline .github/workflows/master.yml:236]: typo

File: .github/workflows/master.yml [hunk 2]
```diff
@@ -161,6 +174,326 @@ jobs:
...
```
Comment (ByronHsu) [inline .github/workflows/master.yml:373]: typo

File: .github/workflows/master.yml (hunk 1 above)
Comment (KUAN-HSUN-LI) [reply .github/workflows/master.yml]: OK

File: .github/workflows/master.yml (hunk 2 above)
Comment (KUAN-HSUN-LI) [reply .github/workflows/master.yml]: OK

**Comment by KUAN-HSUN-LI** [comment]
@ByronHsu Thanks for your review!

**Comment by ByronHsu** [comment]
LGTM

**Pull request closed without merging**